### ISSUE TITLE
yanglint won't start in some timezones

### ISSUE THREAD

**Issue by samuel-gauthier** [CESNET/libyang#632]
Hi,

We have problems starting yanglint, or our custom application using libyang (0.16.46) here on ubuntu 16.04 and ubuntu 18.04 with the lastest version of the tzdata package (2018e-0ubuntu0.16.04).

Basically, if we set the Europe/Dublin or Africa/Windhoek timezones, yanglint won't start at all.
```
root@ubuntu1604:~/libyang# timedatectl set-timezone Europe/Dublin
root@ubuntu1604:~/libyang# yanglint
libyang[0]: Invalid date "2017-02-20", valid date in format "YYYY-MM-DD" expected.
libyang[0]: Module "yang" parsing failed.
Failed to create context.
root@ubuntu1604:~/libyang# 
root@ubuntu1604:~/libyang# 
root@ubuntu1604:~/libyang# timedatectl set-timezone  Africa/Windhoek
root@ubuntu1604:~/libyang# yanglint
libyang[0]: Invalid date "2016-08-05", valid date in format "YYYY-MM-DD" expected.
libyang[0]: Module "ietf-yang-metadata" parsing failed.
Failed to create context.
root@ubuntu1604:~/libyang# 
root@ubuntu1604:~/libyang# 
root@ubuntu1604:~/libyang# timedatectl set-timezone  Europe/Prague 
root@ubuntu1604:~/libyang# yanglint
> 
root@ubuntu1604:~/libyang# timedatectl set-timezone  Europe/Paris 
root@ubuntu1604:~/libyang# yanglint
> 
```

A quick check show that only the two timezones Europe/Dublin and Africa/Windhoek have the problem. I am a bit puzzled, I must say, any help would be much appreciated.

**Comment by michalvasko** [comment]
Hi,
should be fixed now (by commit a78c92ca29c53741c0c0efad9873d4230e15fe98, I made a typo in the reference).

Regards,
Michal

**Comment by samuel-gauthier** [comment]
It does Michal, thanks a lot.
